ppr-ui/src/composables/mhrInformation/useMhrInformation.ts

```diff
@@ -53,13 +54,28 @@ export const useMhrInformation = () => {
     getMhrTransferHomeOwners.value.forEach(ownerGroup => {
       const { groupId, ...owners } = ownerGroup
       ownerGroups.push({
-        owners: [owners],
+        owners: [owners].filter(owner => owner.action !== ActionTypes.REMOVED),
         groupId: parseInt(groupId),
         type: 'SO'
       })
     })
 
-    return ownerGroups
+    return ownerGroups.filter(group => group.owners.length !== 0)
+  }
+
+  const parseRemovedOwnerGroups = (): any => {
+    const ownerGroups = []
+
+    getMhrTransferHomeOwners.value.forEach(ownerGroup => {
+      const { groupId, ...owners } = ownerGroup
+      ownerGroups.push({
+        owners: [owners].filter(owner => owner.action === ActionTypes.REMOVED),
+        groupId: parseInt(groupId),
+        type: 'SO'
```

Comment:
We mark all deleted owners as Sole Owners?

Reply:
This was copy pasta originally, probably from our MVP approach as SO?
This should be dynamic yes, for both Added and Removed. Will update this.

Reply:
After pulling in your commit, there is a bit more work to do here to figure out the type based on what is being submitted in the payload vs what is in Store.

Reply:
Updated. Bit more work than anticipated.